daemons/based/pacemaker-based.h

```diff
@@ -43,10 +43,20 @@ enum cib_client_flags {
     cib_is_daemon      = (UINT64_C(1) << 12),
 };
 
+/*!
+ * \internal
+ * \enum cib_op_attr
+ * \brief Bit flags for CIB operation attributes
+ */
+enum cib_op_attr {
+    cib_op_attr_none       = 0,         //!< No special attributes
+    cib_op_attr_modifies   = (1 << 1),  //!< Modifies CIB
+    cib_op_attr_privileged = (1 << 2),  //!< Requires privileges
```

Comment:
The stickler in me doesn't like using a verb form in one and an adjective form in another (non-parallel structure). A future commit will introduce `cib_op_attr_transaction` (noun) though, so all bets are off. `*_supports_transaction` is too long, as is `*_needs_privileges`.

Also the meaning of `cib_op_transaction` (if we dropped "_attr") would be too easily confused.

Reply:
Agreed